railties/test/generators/plugin_generator_test.rb

```diff
@@ -44,6 +44,21 @@ def test_invalid_plugin_name_raises_an_error
     assert_equal "Invalid plugin name Digest, constant Digest is already in use. Please choose another plugin name.\n", content
   end
 
+
+  def test_valid_plugin_name_not_raising_an_error
+    content = capture(:stderr){ run_generator [File.join(destination_root, "my_plugin-extension")] }
+    assert_equal "", content
```

Comment:
Alright, I will remove these lines and add some other assertions to cover mix names with underscores and dashes.